### ISSUE TITLE
Sometimes Internal Keyboard Not Working

### ISSUE THREAD

**Issue by smileynet000** [linuxmint/cinnamon#11954]
### Distribution

Mint 21.2

### Package version

??

### Graphics hardware in use

??

### Frequency

Quite often

### Bug description

The number keys except 5 and 6 do not work at certain times, in the regular number keys section. I seem to have found a test with showkey, showing that the hardware doesn't seem to be to blame. So it's cinnamon? This happens sometimes. Unfortunately, without the onscreen keyboard or plugging into another keyboard, I often cannot even log onto the system, including the tty. Occasionally, this will also happen to the external keyboard, making the only way to logon, the onscreen keyboard. That keyboard is immune it the problem, so far. It appears to happen randomly, but when it happens, it can take a large number of reboots, at best, to make the system usable. I was almost going to replace the keyboard, but I now believe it's a software issue. This is another big one for me. I love the OS, if you can get the bugs out of it. However, it is better than windows, so I'm staying on linux for now.

### Steps to reproduce

Happens automatically, randomly.

### Expected behavior

Works properly

### Additional information

It does affect the tty too, but I'm still thinking it could? be cinnamon at fault???

**Comment by SonicandTailsCD** [comment]
No, it's not Cinnamon. It's possible that your keyboard may be damaged. Try making a Windows USB and booting off it, then open a command prompt with `Shift + F10`. Type `notepad`, hit enter. Type all kinds of random letters/numbers. If it works, maybe Linux is corrupt?? But if Windows fails, you may need to get a new keyboard to solve the problem ;)

**Comment by smileynet000** [comment]
I now suspect it is a keyboard issue, rather than cinnamon. But because of the logitech issue, couldn't be sure yet.

http://smileynetmain.createaforum.com/other/linux-mint-21-2-keyboard-not-working/

**Comment by SurinameClubcard** [comment]
Using LMDE6, I'm having the same issue. I suspect it has something to do with the "Super"-key (Windows logo) and/or gTile. Indeed, keys 5 and 6 are working! To get things working again, I'm using a second computer, doing a remote SSH and issue a reboot. :-(

**Comment by inetch** [comment]
I have the same issue - the numbers sometimes do not work on the main part of the keyboard, except of the keys 5 and 6. I use Lenovo laptop, and Cinnamon 22.1.
